### PR TITLE
Changing the License to MIT License

### LINKED ISSUE #100: Change LICENSE to MIT

### PR DESCRIPTION
**Pull request by Aayush3014**
Fixes #100

I have changed the License to MIT as asked in the mentioned issue by modifying the License File.
Check everything which applies

- [ ] I have added the issue number for which this pull request is created.

### CODE REVIEW

**Comment by Aayush3014** [comment]
Hey, I have done the changes as asked. Please review the modification and let me Know if i can help in some other issues.

**Comment by arshadkazmi42** [comment]
@Aayush3014 Thank you for working on this, but there is another PR already created #101  for changing this license before you. So we will have to accept that PR. 

We will be closing this PR.

File: LICENSE [hunk 1]
```diff
@@ -1,27 +1,21 @@
-Copyright (c) 2015, First Issues
-All rights reserved.
+MIT License
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
+Copyright (c) 2023 Ayush
```
Comment (arshadkazmi42) [inline LICENSE:3]: `Ayush` => `First Issues`

**Review by arshadkazmi42 — COMMENTED** [review]
Looks good. can you fix the one request change? Rest looks good

**Comment by arshadkazmi42** [comment]
@Aayush3014 we haven't received any updates on the other pull request so I am re-opening your PR. If you could make the requested changes I can get this merged

**Comment by Aayush3014** [comment]
Yeah sure working on it.

**Review by arshadkazmi42 — APPROVED** [review]
looks good thank you

**Pull request merged**